news/4910.bugfix

```diff
@@ -0,0 +1,6 @@
+Workaround for a Windows issue (encountered for example on Windows 10 machines,
```

Comment:
Please rewrite this fragment as per the documentation: https://pip.pypa.io/en/latest/development/#adding-a-news-entry

> In order to maintain a consistent style in the NEWS.rst file, it is preferred to keep the news entry to the point, in sentence case, shorter than 80 characters and in an imperative tone -- an entry should complete the sentence "This change will ...". In rare cases, where one line is not enough, use a summary line in an imperative tone followed by a blank line separating it from a description of the feature/change in one or more paragraphs, each wrapped at 80 characters. Remember that a news entry is meant for end users and should only contain details relevant to an end user.